.env

```diff
@@ -0,0 +1 @@
+REACT_APP_GOOGLE_CLIENT_ID = 221918854202-jm934nkta8u67vk9d4ic3msmqrog46ee.apps.googleusercontent.com
```

Comment:
In order to protect your client id you can delete this file and run the project like this:
npm run start REACT_APP_GOOGLE_CLIENT_ID = your_client_id